src/routes/embedded/auth-request/connect/connect-settings.view.tsx

```diff
@@ -30,7 +29,7 @@ export function WalletPermissionsRequestEmbeddedView() {
   const { url = "", permissions: authRequestPermissions = [], appInfo = {}, gateway } = authRequest;
 
   const [requestedPermissions, setRequestedPermissions] = useStorage<PermissionType[]>({
-    key: `requested_permissions_${url}`,
```

Comment:
This stored value is only used to move some data across the 3 different "connect" screens, so there's no need to add `url` in the key because there will never be 2 values at the same time.